### ISSUE TITLE
Hypospermatogenesis

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9824]
**Preferred term label:**
Hypospermatogenesis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Elongated spermatids are present in the biopsy but spermatogenesis is not quantitatively normal. This term applies to biopsies with some tubules showing diminished amounts of qualitatively complete spermatogenesis in combination with a vast majority of tubules that lack germ cells.

**Parent term (use hpo.jax.org/app)**
HP:0011961

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399771

**Your nano-attribution (ORCID)**
ORPHA: 399771

**Comment by pnrobinson** [comment]
@MagoWyr 

I am adapting the definition a little to give it more context for nonspecialists

Def: A morphological abnormality visible upon biopsy of the germinal epithelium of the testis in which the cellularity of the germinal epithelium is reduced in general, so that all stages of germ cells (spermatogonia, spermatocytes, and spermatids) are present but reduced in number. This term applies to biopsies with some tubules showing diminished amounts of qualitatively complete spermatogenesis in combination with a vast majority of tubules that lack germ cells.
PMID: [21346828](https://pubmed.ncbi.nlm.nih.gov/21346828)

I think that Abnormal spermatogenesis HP:0008669 is the proper parent term.

Please reopen if this needs revision

**Comment by MagoWyr** [comment]
I am fine with this.
________________________________

**Comment by MagoWyr** [comment]
I think this might be an even better definition:

" All stages of spermatogenesis are present but reduced to a varying degree. This definition also includes varying patterns that can result in some tubules showing an epithelium containing Sertoli cells only. The presence of all stages through to the Sd2 stage spermatids indicates the capacity of the testis to generate sperm." (PMID: 16887924)

Because in the definition you sent is says that germ cells are overall reduced, which is not necessarily the case. The number of spermatogonial cells can still be normal with only a reduced number of round and elongating spermatids.
________________________________
